### DISCUSSION TITLE
Please add this lyric client *Better Lyrics (Lyrics for Youtube Music)* its a chrome extention and its the best

### DISCUSSION THREAD

**Discussion by Zippu-4249** [pear-devs/pear-desktop#2544] [Ideas]
PLEASE ADD BETTER LYRICS EXTENTION FROM CHROME
![image](https://github.com/user-attachments/assets/79b6a336-041b-40d0-8350-37877705f4ad)

The sync lyric option doesn't work for 40% of the song and the animations text isn't perfectly synced and not clean 

(in the better lyrics part I have also used an extention called Themesong for youtube music for the album theme and everything )

take a look at this Better lyric example
![image](https://github.com/user-attachments/assets/8e41c9ba-9d4c-40a0-bc7d-d5eed94c9bf9)

and now compare it with Synced lyrics ( Never mind there wasn't a lyrics for this song - _ - )
![image](https://github.com/user-attachments/assets/8d6933b6-7d31-4dcb-8163-1acb0f703b19)

Lets try another example of Better lyrics
![image](https://github.com/user-attachments/assets/b4baf45d-a912-4805-bece-8a6c2552d6f6)

And compare it with Synced lyrics
![image](https://github.com/user-attachments/assets/f7cebbd2-b0e4-4bd5-b9bd-9d11eb55011d)

More examples *Better lyrics*
![image](https://github.com/user-attachments/assets/f48522f4-5867-443d-8e5c-b981b16ee788)

More examples *Synced lyrics*
![image](https://github.com/user-attachments/assets/053f3449-5bf2-4532-8e55-9f4296434f2d)

More examples *Better lyrics*
![image](https://github.com/user-attachments/assets/2ec0a77d-c65e-4ca2-bec9-32e63c5a12bb)

More examples *Synced lyrics*
![image](https://github.com/user-attachments/assets/6ec167f4-77de-49d6-8c20-e5b934b905e5)

Devs please just add this feature

**Comment by JellyBrick** [comment]
Can't. Electron does not support Chrome extensions. (While theoretically possible, it behaves differently than Chrome and is missing a number of features)

Additionally, the option to import lyrics from multiple lyric providers is already being implemented. See #2383

**Reply by Zippu-4249** [reply]
Can you make the text and effect like that it looks clean and its perfectly synced please try smth

**Reply by Zippu-4249** [reply]
It would be pretty nice if there was a discord server to discuss on

**Reply by Zippu-4249** [reply]
there is no option to change between diffrent lyrics provider

**Reply by Lockszmith-GH** [reply]
no option yet, it is being developed
